UnityProject/Assets/Scripts/Items/Implants/Organs/Heart.cs

```diff
@@ -44,6 +44,12 @@ public class Heart : BodyPartFunctionality
 
 	private bool alarmedForInternalBleeding = false;
 
+	[SerializeField]
+	private Reagent salt;
+
+	[SerializeField]
+	private float danagerSaltLevel = 2f; //in %
```

Comment:
Surely this was meant to be "danger" and not "danager",  right?

Reply:
i forgor 💀

Reply:
dammage moment.

Reply:
Done 👍

Reply:
Arma 3's scripting language to this day has setDamage and setDammage commands